### DISCUSSION TITLE
関係者 自己紹介スレ

### DISCUSSION THREAD

**Discussion by ko1** [rubima/magazine.rubyist.net#364] [未分類]
ここに編集者の自己紹介を書いてください。意気込みとか。

例：
Heroku, Inc. のささだです。
普段は Ruby の開発をしています。
@ko1 に編集に誘われました。
Ruby に興味があるので、るびまの編集に協力させて下さい。
あまり時間がないので、リリース前のチェックなどができればと思っています。
よろしくお願いします。

**Comment by SugHimsi** [comment]
sughimsi (すぎむし) です。るびま編集会議にふらっと来ました。
今後は記事のプルーフリーディングなど、できるところで貢献したいと思います。
よろしくお願いします。

**Comment by ore-public** [comment]
小西です。
普段はRailsでアプリケーションの開発しています。
18号か19号くらいの時に、るびま止めるかみたいな話が出たのがきっかけで参加しました。
ここ数年、ほとんど出来てないですが、余裕あるときはプルーフリーディングとか
出来る範囲でやりたいと思っています。

**Comment by miyohide** [comment]
みよひでです。普段はいわゆるSIerでRubyの布教を行っております。

読者さんに興味を持って頂けるような記事を集め、スムーズに掲載するお手伝いをさせていただければと思っております。小心者な自分ですが「るびま編集から参りました」といえば、すごい人とお話ができて交流できるというのは、るびまが持つバリューだと思っています。

これからも、このバリューを最大限活用しつつ、記事を載せていければと思っております。よろしくお願いします。

**Comment by hokkai7go** [comment]
すがいです。よく使うハンドルネームは hokkai7go です。

カンファレンスのレポートをよく書いていました。その流れでるびまにもするりと入ってきました。
いろいろ課題があると認識しているので、るびまを良くするために何が出来るかなと考えています。
よろしくお願いします。

**Comment by kurotaky** [comment]
はじめまして、ペパボのくろたき(@kurotaky)です。

Yokohama.rbでみよひでさんに声をかけていただきました。
るびまはいつも読者として楽しんでいましたが、
今回(51号)からできる範囲で協力したいと思っています。よろしくお願いいたします!

**Reply by kurotaky** [reply]
2023/03/04開催の[鹿児島Ruby会議02](https://k-ruby.com/kagoshima-rubykaigi02/) についてどこかで寄稿したいです！ 📝

**Comment by zunda** [comment]
zundaです

最近Herokuのサポートになりました。体調を崩してからあんまり編集できていないのですが、そろそろ復活したいです。よろしくお願いします！

**Comment by gunjisatoshi** [comment]
書いていないことに気がついた！！１

郡司です。

普段は Excel を使って仕事をしてますが、たまに Excel から tsv を生成して Ruby でデータの処理をしたりしています。

るびまとの関わりはもう何がきっかけだったかは忘れてしまいましたが、読者としては 2005 年くらいから、記事の執筆者としては大江戸 01 のレポートあたりから、編集者としては yugui さんの Hotlinks のテープ起こしあたりから関わっているような気がします。

るびまには最近はあんまり関われていませんが、これからもできる範囲で参加したいと思いますのでよろしくお願いします！

**Comment by dogatana** [comment]
はじめまして、市田です。

るびまのKindle化の流れでやってきました。

普段は組み込み機器の測定データの解析、Excel ＋ Tk で業務の改善などにRubyを利用しています。
Ruby は汎用プログラミング言語として使っているので、Rails方面は全く未知の世界です。

よろしくお願いします。

**Comment by metanest** [comment]
きしもとです。
ええと、書き込みとか出来るのかな？

**Comment by bake0937** [comment]
はじめまして、岡部(@bake0937)です

普段は、Ruby、PHP、JavaScriptを書いております
先日、るびまで福岡Ruby会議02のイベントレポートを書きました
るびまのお手伝いについて、 @kurotaky さんからお誘い頂き参加させて頂くことになりました。
自分にできるところからになりますが、少しでもお手伝いできたらと思います。
よろしくお願いいたします。

**Comment by 3150** [comment]
はじめまして、金築（@takaaki-kanetsuki)です。
Ruby-jpのSlackの#rubimaからやって参りました 🏃 

現在大学在学中で、Web系の企業でインターンでRubyを書いています。
ゆくゆくは東南アジアに移住したい気持ちが強く、海外のRubyConfにも参加しており、そのイベントレポートを寄稿させていただけたらと思います！　よろしくお願い致します 🙇

**Comment by hsbt** [comment]
せっかくなので discussion にしてみました。

@hsbt 柴田です。

主にるびまのサーバーというか、配信をいい感じにする作業や GitHub やコードベースの手入れをしています。たまに記事の編集もやります。

**Reply by hsbt** [reply]
pin にして discussion に大きく表示してみたけどなんかすごくなった

**Comment by neko314** [comment]
金子（ @neko314 ）と申します。

2023年時点でRubyist歴5年とか6年くらいなんですが、私にとってのるびまは歴史的資料の側面が強く、自分がRubyと出会うずいぶん前の様子を知れたり、活躍されているみなさまのことを詳しく知れたり、号を追うとRubyやRuby周辺の変遷を感じれたりして大変興味深く読んでいました。また「難易度：高」な巻頭言のシリーズが好きです。

直近ではRuby30周年記念イベントのレポートを担当します。よろしくお願いします。

**Comment by emorima** [comment]
@emorima 江森です。

Rails Girls開催の記事書いたり、書いてーのお願いしてきました。

大好きなるびまのもうちょっと役に立てるように、動いていきたいと思ってます！

**Comment by yuki-snow1823** [comment]
ほりゆう( @yuki-snow1823 ) と申します。

普段Rubyで業務をしていて、Kaigi on Rails2023と2024のオーガナイザーを務めました。
妻とurawa.rbの主催などもしています。

Rubyist Hotlinks のコーナーが特に好きで、様々なRubyistの方を知るとても良いきっかけになっています。

@neko314 さんの活動を見て何かお力になりたいなと考えていたのですが、
丁度、次に入社するタイミングまで時間ができたので参加させていただきました。

どうぞよろしくお願いいたします。

**Comment by sezemiadmin** [comment]
ひろせ (@sezemiadmin) です !

2025/12/20 開催の Ruby リリース 30 周年パーティで @neko314 さんの発表を聞いて、 るびま の編集に興味を持ちました !

普段は採用広報の仕事をしていて、テックブログの編集や記事を書いたりしています。 Ruby はほとんど経験がなく、前職で [研修レポートの仕事](https://www.seplus.jp/dokushuzemi/blog/2022/06/learn_rails_by_author.html) をした際、 `scaffold` でワイワイしたぐらいです。 

 るびま との出会いは Hotlinks の須藤さんのインタビューで、それから折につけ、るびま を読んでいました。

なにかしら、編集や書いたりで手伝えることがあればと思っています !
